extension/chrome/elements/attachment.ts

```diff
@@ -79,9 +79,10 @@ export class AttachmentDownloadView extends View {
       return;
     }
     $('#type').text(this.type || 'unknown type');
-    $('#name').text(this.name || 'noname');
+    const attachmentName = decodeURIComponent(this.name || 'noname');
```

Comment:
Are you sure we need this extra decoding? 
I sent a message with an attachment name `what's_up%253F.txt`
When downloaded, the file name became `what's_up%3F.txt`
(I committed the message 182263bf9f105adf to the repo)
Please add a test that the downloaded file name matches the initial one.

Reply:
You are right. We actually don't need this encoding.